client/src/app/site/polls/components/base-poll-detail.component.ts

```diff
@@ -213,6 +192,10 @@ export abstract class BasePollDetailComponent extends BaseModelContextComponent
                 {
                     idField: 'content_object_id'
                 },
+                {
+                    idField: 'voted_ids',
+                    fieldset: 'shortName'
+                },
                 {
                     idField: 'option_ids',
                     follow: [{ idField: 'vote_ids' }, { idField: 'content_object_id' }]
```

Comment:
I have not noticed before, that you want to get the structure-level, the number and the vote-weight of a user.
To have this, you could re-write this line to:
```ts
                    follow: [
                        {
                            idField: 'vote_ids',
                            follow: [
                                {
                                    idField: 'user_id',
                                    fieldset: 'shortName',
                                    additionalFields: [
                                        { templateField: 'structure_level_$' },
                                        { templateField: 'vote_weight_$' },
                                        { templateField: 'number_$' }
                                    ]
                                }
                            ]
                        },
                        { idField: 'content_object_id' }
                    ]
```